server/konnectors/orange_mobile.js

```diff
@@ -31,6 +31,21 @@ const connector = module.exports = baseKonnector.createNew({
   },
 
```

Comment:
We may add the orange_mobile connector into 'telecom' category, what do you think ?